README.md

```diff
@@ -0,0 +1,366 @@
+#### Get orders (USER)
+
+```code
+  GET "/api/orders"
+```
+get user orders
+*****
+#### Patch status (ADMIN)
+
+```code
+  PATCH "/api/orders/{id}"
+```
+body:
+| Parameter  | Description                       |
+| :--------  | :-------------------------------- |
+| `id`      | **Required** id of order|
+
+```code
+  status : "new status"
+```
+change status of order
+*****
+#### Get Order Items (USER)
+
+```code
+  GET "/api/orders/{orderId}/items"
+```
+body:
+| Parameter  | Description                       |
+| :--------  | :-------------------------------- |
+| `orderId`      | **Required** order id |
+
+get order items by order id
+*****
+#### Get Order Item in order (USER)
+
+```code
+  GET "/api/orders{orderId}/items/{itemId}"
+```
+body:
+| Parameter  | Description                       |
+| :--------  | :-------------------------------- |
+| `orderId`      | **Required** order id |
+| `itemId`      | **Required** order item id |
+
+get order item by order id and order item id
+
+*****
+#### add item to cart (USER, ADMIN)
+
+```code
+  POST "/api/cart"
+```
+body:
+| Parameter  | Description                       |
+| :--------  | :-------------------------------- |
+| `bookId`      | **Required** id of book |
+| `quantity`      | **Required** quantity of books |
+
+```code
+{
+    "bookId":1,
+    "quantity":2
+}
+```
+post book to cart
+
+*******
+#### Get cart (USER, ADMIN)
+
+```code
+  GET "/api/cart"
+```
+
+get user cart
+
+*******
+#### update cartItem (USER, ADMIN)
+
+```code
+  PUT "/api/cart/cart-items/{cartItemId}""
+```
+body:
+| Parameter  | Description                       |
+| :--------  | :-------------------------------- |
+| `cartItemId`      | **Required** id of cart |
+| `bookId`      | **Required** id of book |
+| `quantity`      | **Required** quantity of books |
+
+```code
+{
+    "bookId":1,
+    "quantity":2
+}
+```
+
+update cart item data
+
+*******
+#### Delete cartItem (USER, ADMIN)
+
+```code
+  DELETE "/api/cart/cart-items/{cartItemId}""
+```
+body:
+| Parameter  | Description                       |
+| :--------  | :-------------------------------- |
+| `cartItemId`      | **Required** id of cart |
+
+delete cartItem from cart
+
+*******
+## 📜How project was developed 📜
+![App Screenshot](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExYW9lNWU1N3plaHdvM2NpaDBoYmZhbjA4N2ozeHd5cnN1YXg2NHZsaiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/UQ84N6dIWObZWILqsE/giphy.gif)
+
+It was my first back-end project, and it was very interesting to implement it, and during the development I encountered some blockers, but thanks to my mentors I was able to overcome them, and I am very grateful to them.
+
+#### spring security
+when adding a security token, namely jwt token, there was a problem that the user could not submit a request on the token because the Expiration in the token was not set correctly and the user could not log in, but together with the mentor we found the reason in the Value annotation and fixed it
```

Comment:
```suggestion
When adding a security token, namely jwt token, there was a problem that the user could not submit a request on the token because the Expiration in the token was not set correctly and the user could not log in, but together with the mentor we found the reason in the Value annotation and fixed it
```